resources/sdk/purecloudjavascript/templates/ApiClient.mustache

```diff
@@ -1,4 +1,5 @@
 import superagent from 'superagent';
+import Configuration from './configuration.js';
 
```

Comment:
Almost forgot, can you add logging to the various authorization functions as well please.

Reply:
Added loggings to authorization functions.